examples/all-clusters-app/linux/AllClustersCommandDelegate.cpp

```diff
@@ -162,7 +162,8 @@ void AllClustersAppCommandHandler::OnRebootSignalHandler(BootReasonType bootReas
 {
     if (ConfigurationMgr().StoreBootReason(static_cast<uint32_t>(bootReason)) != CHIP_NO_ERROR)
     {
-        Server::GetInstance().DispatchShutDownAndStopEventLoop();
+        Server::GetInstance().GenerateShutDownEvent();
```

Comment:
btw, the condition here seems inverted - the error message is generated when there's no error...

Reply:
So it does.  I'll fix that in a separate PR, just in case.

Reply:
@Damian-Nordic https://github.com/project-chip/connectedhomeip/pull/25154